### DISCUSSION TITLE
I can't submit issues to carbon filed

### DISCUSSION THREAD

**Discussion by digikordeveloper** [community/community#146180] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Every time I try to submit I am getting an error page with no explanation, can someone help?

**Comment by shinybrightstar** [comment]
Hi @digikordeveloper , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
